collectors/apps.plugin/apps_plugin.c

```diff
@@ -3803,6 +3813,8 @@ static void send_charts_updates_to_netdata(struct target *root, const char *type
             if (unlikely(w->exposed))
                 fprintf(stdout, "DIMENSION %s '' absolute 1 1\n", w->name);
         }
+        if (!strcmp("apps", type))
+            fprintf(stdout, "VARIABLE fdperc '' absolute 1 1\n");
```

Comment:
Over here I never recall adding this, maybe I should remove it (might mislooked it somehow)